auto_nag/bzcleaner.py

```diff
@@ -275,6 +278,10 @@ def amend_bzparams(self, params, bug_ids):
             n = utils.get_last_field_num(params)
             params.update({'f' + n: 'bug_group', 'o' + n: 'isempty'})
 
+        if self.only_defects():
```

Comment:
maybe you could use this opportunity to do the same for task and enhancement as we will use that for sure